jiant/huggingface_transformers_interface/modules.py

```diff
@@ -358,8 +361,8 @@ def get_pretrained_lm_head(self):
             self.input_module, cache_dir=self.cache_dir
         )
         lm_head = model_with_lm_head.lm_head
-        lm_head.predictions.decoder.weight = self.model.embeddings.word_embeddings.weight
-        return nn.Sequential(lm_head, nn.LogSoftmax(dim=-1))
+        lm_head.decoder.weight = self.model.embeddings.word_embeddings.weight
```

Comment:
thank you!

Reply:
The  LM decoder weights being returned in the head was not initialized from the pretrained model, which was a bug.

Reply:
Thanks for the clarification. Can you also spell out for me what the the motivation was for removing `nn.Sequential()` / `nn.LogSoftmax(dim=-1)` on the neighboring line? (related to [Sam's question](https://github.com/nyu-mll/jiant/pull/1030/files#r396128626))

Reply:
Thanks — I see that after your changes `get_pretrained_lm_head()` will return `lm_head` without applying `nn.Sequential()` and `nn.LogSoftmax()` for these `HuggingfaceTransformersEmbedderModule`s:
- `RobertaEmbedderModule`
- `XLNetEmbedderModule`
- `OpenAIGPTEmbedderModule`
- `GPT2EmbedderModule`
- `TransfoXLEmbedderModule`

It looks like `get_pretrained_lm_head()` will continue to `return nn.Sequential(lm_head, nn.LogSoftmax(dim=-1))` for these `HuggingfaceTransformersEmbedderModule`s:
- `AlbertEmbedderModule`
- `BertEmbedderModule`
- `XLMEmbedderModule`

For consistency can you update the remaining `HuggingfaceTransformersEmbedderModule`s?